README.md

```diff
@@ -15,10 +15,9 @@
 
 <hr />
 
-
 D++ is a lightweight and efficient library for **Discord** written in **modern C++**, covering as much of the API specification as possible with an **incredibly small memory footprint** even when caching large amounts of data.
 
-### Library features:
+### Library Features
```

Comment:
Semi-Colon should probably stay as it's a list

Reply:
Felt inconsistent with other headers in there, but open to it if it's wanted.

Reply:
on this i disagree, dont usually put colon in a title

Reply:
I'm happy for it to not have a semi-colon if that's right :)